manifests/rhoai/shared/odhdashboardconfig/odhdashboardconfig.yaml

```diff
@@ -1,41 +1,14 @@
-# ODHDashboardConfig to enable the notebook-controller feature and add notebook sizes
+# Read the README before modifying
 apiVersion: opendatahub.io/v1alpha
 kind: OdhDashboardConfig
 metadata:
-  creationTimestamp: null
   name: odh-dashboard-config
+  annotations:
+    opendatahub.io/managed: "false"
 spec:
   dashboardConfig:
-    disableBYONImageStream: false
-    disableClusterManager: false
-    disableISVBadges: false
-    disableInfo: false
-    disableSupport: false
+    # Overrides only -- see blankDashboardCR
     disableTracking: false
-    enablement: true
-    disableHome: false
-    disableProjects: false
-    disablePipelines: false
-    disableModelServing: false
-    disableProjectScoped: false
-    disableProjectSharing: false
-    disableCustomServingRuntimes: false
-    disableTrustyBiasMetrics: false
-    disablePerformanceMetrics: false
-    disableAcceleratorProfiles: false
-    disableKServe: false
-    disableKServeAuth: false
-    disableKServeMetrics: false
-    disableModelMesh: false
-    disableDistributedWorkloads: false
-    disableModelRegistry: false
-    disableModelRegistrySecureDB: false
-    disableServingRuntimeParams: false
-    disableStorageClasses: false
-    disableNIMModelServing: false
-  groupsConfig:
-    adminGroups: "$(admin_groups)"
-    allowedGroups: "system:authenticated"
```

Comment:
Request has been made to the Operator team about the removal of this

Tracked here: https://issues.redhat.com/browse/RHOAIENG-23601